tools/run-fbp-tests

```diff
@@ -269,6 +270,7 @@ def run_tests(args):
                 logging.warning("Dependence test %s for test %s failed. Skipping it",
                         curr.depends, curr.test_file)
                 status["skipped"] += 1
+                logging.info("FAIL: %s" % path["file"])
```

Comment:
@barbieri That's OK. The case should be blocked by fail case. However, by reading ptest result definition (http://www.yoctoproject.org/docs/2.0/dev-manual/dev-manual.html#testing-packages-with-ptest), it seems BLOCK is not in the list. I can amend the patch and log the result as SKIP. Thanks.